Sources/SwiftUILogger/Public/SwiftUILogger.swift

```diff
@@ -84,7 +87,8 @@ open class SwiftUILogger: ObservableObject {
             message: String,
             error: Error? = nil,
             _ file: StaticString = #fileID,
-            _ line: Int = #line
+            _ line: Int = #line,
+            _ tags: [AnyHashable] = []
```

Comment:
Let's move this parameter right after the error parameter. File and line parameters are always autofilled. 

Also let's expose the tags parameter name and remove the _.

Reply:
@0xLeif Alright, done here and all other places where same situation